templates/forms/default/field.html.twig

```diff
@@ -2,6 +2,8 @@
 {% set toggleableChecked = field.toggleable and (originalValue is not null and originalValue is not empty) %}
 {% set isDisabledToggleable = field.toggleable and not toggleableChecked %}
 {% set value = (is_object(value) or value is null ? field.default : value) %}
+{% set cookie_name = 'forms-' ~ form.name ~ '-' ~ field.name %}
+{% set value = (is_object(value) or value is null ? (get_cookie(cookie_name) is null ? field.default : get_cookie(cookie_name)) : value) %}
```

Comment:
@hughbris I had to remove the check for `is_object()` as it breaks some other fields taking objects as values. Was there any reason why you had it there?

Reply:
I'm sorry I can't remember precisely but I have a feeling something failed without that check. You might have to test on some object-type fields, can't even recall what they were :/

Reply:
@hughbris Please check the next beta.5 when it gets out.